js/index.js

```diff
@@ -20,7 +20,7 @@ INSTRUCTIONS:
     project, when tests are failing, 
```

Comment:
in index.html line 34 <br> tag closed improperly, spelling error, should be </br> not </brz>

Reply:
- [ ] Kindly add a meaningful code and review that; this project aims to learn how to do a review and learn HTML and CSS codes. In giving a code review, please be polite and clear about the inline comment you give feedback on, and explain why it required changing aspects.

Reply:
The review was polite and clear as well as neutral, it requires changes because the code is incomplete and has a spelling error as mentioned in the review.